src/style.css

```
@@ -0,0 +1,64 @@
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.searchbar1 {
```

```suggestion
.searchbar {
```
зачем там цифра? где-то есть searchbar2?